Orange/widgets/evaluate/owliftcurve.py

```diff
@@ -382,11 +493,30 @@ def set_visual_settings(self, key, value):
         self.visual_settings[key] = value
 
 
-def points_from_results(results, target, clf_index):
-    x, y, thresholds = cumulative_gains_from_results(results, target, clf_index)
-    points = CurveData(x, y, thresholds)
-    hull = CurveData(*convex_hull([(x, y, thresholds)]))
-    return PointsAndHull(points, hull)
+def points_from_results(results, target, clf_index, curve_type):
+    func = precision_recall_from_results \
+        if curve_type == CurveTypes.PrecisionRecall \
+        else cumulative_gains_from_results
+    x, y, thresholds = func(results, target, clf_index)
+    return CurveData(x, y, thresholds)
+
+
+def precision_recall_from_results(results, target, clf_idx):
+    y_true = results.actual
+    classes = np.unique(results.actual)
+    if len(classes) > 2:
+        y_true = label_binarize(y_true, classes=sorted(classes))
+        y_true = y_true[:, target]
+    scores = results.probabilities[clf_idx][:, target]
+    precision, recall, thresholds = precision_recall_curve(y_true, scores)
+
+    if thresholds[-1] < 1:
```

Comment:
Perhaps add a comment here, explain that scikit will add this point, and sometimes we have to remove the point and somehow we need to add data.